### ISSUE TITLE
Trim DENYLIST.s390x

### ISSUE THREAD

**Issue by iii-i** [libbpf/ci#129]
All tests listed in https://github.com/kernel-patches/bpf/blob/bpf-next_base/ci/vmtest/configs/DENYLIST.s390x pass in my development setup:

```
# ./test_progs -t deny_namespace,tc_redirect/tc_redirect_dtime,lru_bug,dummy_st_ops/dummy_init_ptr_arg,fexit_bpf2bpf,tailcalls,trace_ext,xdp_bpf2bpf,xdp_metadata
[...]
Summary: 9/34 PASSED, 0 SKIPPED, 0 FAILED
```

In particular, `dummy_st_ops/dummy_init_ptr_arg` was fixed by `7ded842b356d s390/bpf: Fix bpf_plt pointer arithmetic`; unfortunately, the others don't have a bug report next to them. Should we empty this file? Or is there a list of bugs associated with the other entries somewhere else?

**Comment by chantra** [comment]
The source of this file is https://github.com/kernel-patches/vmtest/blob/master/ci/vmtest/configs/DENYLIST.s390x

unfortunately, the best option if there is no clear comment as to why a test was added, is to check the associated commit in https://github.com/kernel-patches/vmtest/commits/master/ci/vmtest/configs/DENYLIST.s390x .

> Should we empty this file?

SGTM, but we would need those to pass on both bpf and bpf-next branches as we may run the tests on one or the other branch. cc @anakryiko

**Comment by anakryiko** [comment]
> > Should we empty this file?
> 
> SGTM, but we would need those to pass on both bpf and bpf-next branches as we may run the tests on one or the other branch. cc @anakryiko

yep, we should wait for bpf-next and bpf trees to converge before landing

**Comment by iii-i** [comment]
Sounds good. Please ping me if they start failing again after that; I will investigate.
